### PR TITLE
Addresses #198, issue with DictVectorizer

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: tests/test_sklearn_dict_vectorizer_converter.py [hunk 1]
```diff
@@ -21,24 +21,19 @@ def test_model_dict_vectorizer(self):
...
-            model,
-            model_onnx,
+            data, model, model_onnx,
             basename="SklearnDictVectorizer-OneOff-SkipDim1",
             allow_failure="StrictVersion(onnxruntime.__version__)"
                           " <= StrictVersion('0.1.3') or "
                           "StrictVersion(onnx.__version__)"
-                          " < StrictVersion('1.3.0')",
-        )
+                          " < StrictVersion('1.3.0')")
```
Comment (prabhat00155) [inline tests/test_sklearn_dict_vectorizer_converter.py:36]: Why these changes, I think the previous version is better for adding new arguments to dump_data_and_model(), which would avoid modifying existing lines.

Comment (xadupre) [reply tests/test_sklearn_dict_vectorizer_converter.py:36]: Sorry, my screen is small :)
